src/data/data1d/filter1d/signalProcessing.ts

```diff
@@ -0,0 +1,83 @@
+import * as Filters from 'ml-signal-processing/src/filters/filters';
```

Comment:
@hamed-musallam This is your bug. you shouldn't be deep-importing from this package

Reply:
Thanks !!! @targos